go/teams/service_helper.go

```diff
@@ -303,6 +304,56 @@ func InviteEmailMember(ctx context.Context, g *libkb.GlobalContext, teamname, em
 	return t.InviteEmailMember(ctx, email, role)
 }
 
+func AddEmailsBulk(ctx context.Context, g *libkb.GlobalContext, teamname, emails string, role keybase1.TeamRole) error {
+	t, err := GetForTeamManagementByStringName(ctx, g, teamname, true)
+	if err != nil {
+		return err
+	}
+
+	emailList := splitBulk(emails)
+	g.Log.CDebugf(ctx, "team %s: bulk email invite count: %d", teamname, len(emailList))
+
+	var invites []SCTeamInvite
+	for _, e := range splitBulk(emails) {
```

Comment:
`splitBulk(emails)` can be `emailList` in the loop

Reply:
argh, thanks